pkg/fission-cli/logdb/influxdb.go

```diff
@@ -144,7 +144,7 @@ func (influx InfluxDB) query(query influxdbClient.Query) (*influxdbClient.Respon
 	}
 	// connect to controller first, then controller will redirect our query command
 	// to influxdb and proxy back the db response.
-	queryURL.Path = fmt.Sprintf("/proxy/%s", INFLUXDB)
+	queryURL.Path = fmt.Sprintf("%s/proxy/%s", queryURL.Path, INFLUXDB)
```

Comment:
Please add `path.Clean(fmt.Sprintf.....` to clean the URL path.

Reply:
Okay, i will recommit it, thanks!